src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/Example3.java

```diff
@@ -9,9 +9,11 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltokentext;
 // xdoc section -- start
-public void myTest() {
-    final String quote = """
```

Comment:
Please restore it , and put this file to non compiled resources folder, we have bunch of examples already

Reply:
Put it to the non-complied folder as requested, xml file edited